test/api_test.js

```diff
@@ -102,6 +102,13 @@ suite('API', () => {
       });
     });
 
+    test('list top-level namespaces throws error if not found', async function() {
```

Comment:
This doesn't seem to describe the test -- the test in fact asserts that an error did *not* occur..

Reply:
Sorry for the redundant test. I have removed it in the new commit. @djmitche